dhcp/unikernel.ml

```diff
@@ -0,0 +1,85 @@
+open V1_LWT
+open Lwt
+
+(* IP Configuration, all you need besides dhcpd.conf. *)
+let ipaddr = Ipaddr.V4.of_string_exn "192.168.1.5"
```

Comment:
Agree-- I'd not hold this up for functoria to merge.

If there are things in (eg) ethif example that needed updating for this example, then possible skeleton needs some general TLC again to bring it back up to date.

(Indeed, having just pushed DNS support into mirage/mirage-seal, I'm tempted to suggest we might think about merging these two repos, as that might make it more likely that code will stay up-to-date with latest Mirage library support... But that might not be a good idea, and needs some thought :)